websites/A/AnimeDigitalNetwork/presence.ts

```diff
@@ -35,14 +30,13 @@ presence.on("UpdateData", async () => {
         delete data.endTimestamp;
       }
 
-      if (title !== null && subtitle !== null) {
+      if (t !== null) {
         presence.setActivity(data, !video.paused);
```

Comment:
yeah also you can remove this second argument since it isn't required in this situation.

Reply:
will have to provide proof of working after removing the second argument as well just as a double check

Reply:
![ADN10](https://user-images.githubusercontent.com/42322979/114664965-bc504180-9cfc-11eb-92de-9dcf3a050187.jpg)
![ADN9](https://user-images.githubusercontent.com/42322979/114664968-bce8d800-9cfc-11eb-9a03-86a1ce179e7e.jpg)